dev-lang/vlang/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+
+<pkgmetadata>
+  <!-- Maintainer needed -->
+  <longdescription>
+    LALALA
+  </longdescription>
```

Comment:
This longdescription is a bit short :P 
You can just remove the tag entirely from the metadata file if there is no useful longdescription to add

Reply:
Ahaha what have I done 😅 completely forgot about it!